### ISSUE TITLE
[BUG] Renaming session breaks minidump/screenshot functionalities

### ISSUE THREAD

**Issue by pielgrzym** [byt3bl33d3r/SILENTTRINITY#135]
# Issue Template

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* SILENTTRINITY Version: 0.4.6dev
* OS running the Client: Windows Server 2012
* Python Version Running the Client: the one from shiv
* OS running the TeamServer: kali
* Python version running the Teamserver: the one from shiv

## Expected Behavior

After renaming the session modules relying on file transfers should work.

## Current Behavior

When stager initializes file transfer (sending minidump or screenshot) team server is trying to create gz file in a directory that does not exist. This is because `~/.sh/logs/` contains directories based on Guids and teamserver is crafting a path based on session name instead. Everything works as long as session name and Guid match.

## Failure Information

Team server is trying to reassemble gzip file in path: `/root/.st/logs/lolrenamedsession/minidump_2019_11_04_121802.gz` (note: lolrenamedsession) instead of: `/root/.st/logs/d969f8a6-b813-4129-a676-074a56ad42f2/minidump_2019_11_04_121802.gz`.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. run stager on remote machine
2. rename session
3. try to run `boo/screenshot`, `boo/minidump`.
   ...

### Failure Logs

Please include any relevant log snippets or files here:

**Teamserver log:**
```
2019-11-04 12:20:28,964 2682 Thread-2 - [ERROR] ipcserver.py: wait_for_event - Error occured in subscriber function to Events.JOB_RESULT event, printing traceback
Traceback (most recent call last):
  File "/root/.shiv/st_0d67ffb4-cc7a-421c-8dce-8c6e619ce9c4/site-packages/silenttrinity/core/ipcserver.py", line 71, in wait_for_event
    client.send((topic, sub(msg)))
  File "/root/.shiv/st_0d67ffb4-cc7a-421c-8dce-8c6e619ce9c4/site-packages/silenttrinity/core/teamserver/contexts/sessions.py", line 143, in job_result
    decrypted_job, job_output = session.jobs.decrypt(job_id, data)
  File "/root/.shiv/st_0d67ffb4-cc7a-421c-8dce-8c6e619ce9c4/site-packages/silenttrinity/core/teamserver/jobs.py", line 55, in decrypt
    output = job.module.process(self, output)
  File "/root/.shiv/st_0d67ffb4-cc7a-421c-8dce-8c6e619ce9c4/site-packages/silenttrinity/core/teamserver/modules/boo/minidump.py", line 56, in process
    with open(self.gzip_file, 'ab+') as reassembled_gzip_file:
FileNotFoundError: [Errno 2] No such file or directory: '/root/.st/logs/lolrenamedsession/minidump_2019_11_04_121802.gz'
2019-11-04 12:20:28,965 2860 MainThread - [DEBUG] ipcclient.py: dispatch_event - Received data back from event: Events.JOB_RESULT - ERROR - [Errno 2] No such file or directory: '/root/.st/logs/lolrenamedsession/minidump_2019_11_04_121802.gz'

```

**C2 client output**
```
Error sending job (id: TkdRpIJ0LH) results with https channel: Unable to perform SendJobResults() operation using callback URLs
```

**Comment by glides** [comment]
I believe I have this fixed in this pull request #134 

Or [here](https://github.com/glides/SILENTTRINITY/commit/3214f526207b4b37a9b4f2c81e1139f5361d2bcd)

**Comment by pielgrzym** [comment]
Great work, thanks! I'll test your branch during the weekend :) hopefully @byt3bl33d3r will have some time to review :+1:
